src/App.js

```diff
@@ -1,23 +1,18 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
```

Comment:
We don't actually need this line of code... It was required in previous versions of React, but you're safe to remove it now